source/RevitLookup/Core/ComponentModel/Descriptors/ViewDescriptor.cs

```diff
@@ -72,6 +77,21 @@ ResolveSet ResolveCategoryOverrides()
             return resolveSummary;
         }
         
+        ResolveSet ResolveIsCategoryOverridable()
+        {
+            var categories = context.Settings.Categories;
+            var resolveSummary = new ResolveSet(categories.Size);
+            foreach (Category category in categories)
+            {
+                if (!category.IsVisibleInUI) continue;
```

Comment:
Do we need this check?

Reply:
Actually it is not necessary, it remains from my attempt to get all categories using Enum.GetValues(typeof(BuiltInCategory));

With this check we have 344 categories, without — 351 categories. The hidden one is not special, like Air Systems or Rebar Shape, so I deleted the check in last commit